app/components/discover-topic-list/template.hbs

```diff
@@ -4,7 +4,14 @@
   valueProperty='url'
   tagName='div'
   onchange=(action 'onMultiselectChangeEvent') as |topic isSelected|}}
-  {{discover-topic title=topic.title url=topic.url isSelected=isSelected}}
+  {{discover-topic
+    title=topic.title
+    url=topic.url
+    isSelected=isSelected
+    class='gtm__click-tracking'
+    data-action='Discover'
+    data-label=(concat (if isSelected 'Deselected Topic ' 'Selected Topic ') topic.title)
```

Comment:
I might be misunderstanding something here, but if `isSelected` is true, wouldn't this be a Selected Topic?

Reply:
Ah, nope, just figured it out, this is a click action which means if it's selected, clicking deselects it. Got it. Moving on.